e2e-tests/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "e2e-tests"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+aurae-client = { workspace = true }
+aurae-proto = { workspace = true }
+futures-util = "0.3.26"
+tokio = { workspace = true }
+uuid = { version = "1.2.2", features = ["v4"] }
```

Comment:
I think we use this dep in multiple crates. Let's move it to the workspace level?

Reply:
`futures-util`, `uuid` or both? :)

Reply:
I was thinking uuid, but both it they are used in multiple crates. I've been "reviewing" from my phone for the last week, so it's hard to keep it all in my head and get a full picture of the changes.